standard/bridge-v1/emulator/main.go

```diff
@@ -68,21 +76,30 @@ func main() {
 	// Construct two eth clients and cancel all pending txes on both chains
 	l1Client, err := ethclient.Dial(l1RPCUrl)
 	if err != nil {
-		log.Fatal().Err(err).Msg("failed to dial l1 rpc")
+		logger.Error("failed to dial l1 rpc", "error", err)
+		os.Exit(1)
 	}
 	settlementClient, err := ethclient.Dial(settlementRPCUrl)
 	if err != nil {
-		log.Fatal().Err(err).Msg("failed to dial settlement rpc")
+		logger.Error("failed to dial settlement rpc", "error", err)
+		os.Exit(1)
+	}
+	if err := shared.CancelPendingTxes(ctx, privateKey, l1Client); err != nil {
+		logger.Error("failed to cancel pending L1 transactions")
+		os.Exit(1)
+	}
+	if err := shared.CancelPendingTxes(ctx, privateKey, settlementClient); err != nil {
+		logger.Error("failed to cancel pending settlement transactions")
+		os.Exit(1)
 	}
-	shared.CancelPendingTxes(ctx, privateKey, l1Client)
-	shared.CancelPendingTxes(ctx, privateKey, settlementClient)
 
 	for {
 		// Generate a random amount of wei in [0.01, 10] ETH
 		maxWei := new(big.Int).Mul(big.NewInt(10), big.NewInt(params.Ether))
 		randWeiValue, err := rand.Int(rand.Reader, maxWei)
 		if err != nil {
-			log.Fatal().Err(err).Msg("Failed to generate random value")
+			logger.Error("failed to generate random value", "error", err)
+			continue
```

Comment:
nit: this shouldn't change behavior imo, os.exit seems appropriate for such an odd err being returned